lib/lightning/dashboard_stats.ex

```diff
@@ -74,6 +75,54 @@ defmodule Lightning.DashboardStats do
     }
   end
 
+  @doc """
+  Sorts a list of workflow statistics based on the specified field and direction.
+
+  ## Parameters
+    * `workflow_stats` - A list of WorkflowStats structs to be sorted
+    * `sort_field` - String representing the field to sort by, options include:
+      * `"last_workorder_updated_at"` - Sort by timestamp of the latest work order
+      * `"workorders_count"` - Sort by total count of work orders
+      * `"failed_workorders_count"` - Sort by count of failed work orders
+    * `sort_direction` - String representing sort direction, either "asc" or "desc"
+
+  ## Returns
+    Sorted list of WorkflowStats structs
+
+  ## Examples
+
+      iex> sort_workflow_stats(workflow_stats, "workorders_count", "desc")
+      [%WorkflowStats{workorders_count: 100, ...}, %WorkflowStats{workorders_count: 50, ...}]
+
+      iex> sort_workflow_stats(workflow_stats, "last_workorder_updated_at", "asc")
+      [%WorkflowStats{last_workorder: %{updated_at: ~U[2023-01-01 00:00:00Z]}, ...}, ...]
+  """
+  def sort_workflow_stats(workflow_stats, sort_field, sort_direction) do
+    sorter = get_sort_function(sort_field)
+    sort_order = String.to_existing_atom(sort_direction)
```

Comment:
I believe this should be handled in the `handle_params`, doing the conversion once and everywhere else could deal only with atoms.

Reply:
Thanks for the catch, this is done.

Reply:
maybe missed the commit? (: it's still `sort_direction = params["dir"] || "asc"`

Reply:
After this change we could remove this: 
  defp string_to_atom("asc"), do: :asc
  defp string_to_atom("desc"), do: :desc

Reply:
Oh man, I am sorry I missed pushing my commits. Could you check again ?